src/styles/pages/lp.module.scss

```diff
@@ -0,0 +1,18 @@
+@use '../abstracts' as a;
```

Comment:
If you are not using modules, please remove them.

Reply:
I deleted the line importing mixin.
I also replaced the background color with the variable I added to `_varialbes.scss`